client/components/NewEntryForm.js

```diff
@@ -63,7 +73,8 @@ class NewEntryForm extends Component {
 
 const mapState = (state) => {
   return {
-    userId: state.user.me.id,
+    user: state.user.me,
+    promptSubject: state.prompt.single.subject,
```

Comment:
what's the diff between a prompt and a subject?

Reply:
will the prompt ever have something other than a subject? another string field where there is elaboration/further direction?

Reply:
Yeah, I think we will need to adjust the models. Since you've shown me what the prompts will look like, and there's some pre-determined initial starter text, we might pre-load that into the entry content, or display it in a larger format above the entry content. Let's discuss it this weekend.